simulator/src/main/java/com/hedera/block/simulator/generator/MockBlockStreamManagerImpl.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.simulator.generator;
+
+import com.hedera.hapi.block.stream.Block;
+import javax.inject.Inject;
+
+/** A mock implementation of the BlockStreamManager interface. */
+public class MockBlockStreamManagerImpl implements BlockStreamManager {
```

Comment:
Why use the `Mock` prefix? Can this just be `BlockStreamManagerImpl`?

Reply:
This is not part of the current scope but needed an impl of the BlockStreamManager to make sure that the wiring was working on Dagger and the App.

However that impl will be removed in the next PR, for now is just a placeholder.

Reply:
Sounds good